xcube/core/resampling/rectify.py

```diff
@@ -634,37 +642,52 @@ def _compute_var_image_numpy(
     dst_height = dst_src_ij_images.shape[-2]
     dst_shape = src_var.shape[:-2] + (dst_height, dst_width)
     dst_values = np.full(dst_shape, fill_value, dtype=src_var.dtype)
+    src_bbox = [0, 0, src_var.shape[-2], src_var.shape[-1]]
```

Comment:
```suggestion
    src_bbox = (0, 0, src_var.shape[-2], src_var.shape[-1])
```

I don't understand why numba isn't complaining here. List types should make it fall back into Python mode, `njit` should raise. Why isn't that happening?